### ISSUE TITLE
PLE1205 false-positive with Loguru

### ISSUE THREAD

**Issue by dorschw** [astral-sh/ruff#13390]
When using [loguru](https://github.com/Delgan/loguru), and configuring it as the logger object, the formatting isn't `%` based, but `{}`-based, see [here](https://loguru.readthedocs.io/en/stable/resources/migration.html).
When I write
```python
logger.info("{}", var)
```
a false-positive PLE1205 is raised.

**Comment by MackHalliday** [comment]
I am having this same issue - It's so annoying. Please lmk if you find a workaround.

**Comment by MichaReiser** [comment]
The PLE1205 for detecting logger's is very basic at the moment. All it does is match on the variable name (this does not apply for `logging` calls). This is because Ruff has no type inference capabilities (we're working on it) that allow reasoning about how values flow through the program. This means there's no immediate fix here. 

I recommend you disable `PLE1205` in your project if you're consistently using (`ignore = ["PLE1205"]`) or ignore it on a per file level: `per-file-ignores = { "file.py": ["PLE1205"]}`

**Comment by MackHalliday** [comment]
Thank you for the response :D

**Comment by RazerM** [comment]
I have the same issue with https://github.com/getlogbook/logbook (`{}` formatting), which I think is exacerbated by this rule now being enabled by default.
